go.mod

```diff
@@ -1,10 +1,13 @@
-module github.com/Shelnutt2/db2struct
+module hktk1001/db2struct
```

Comment:
Please revert the mod files (`go.mod` and `go.sum`), the changes here look like they are only from your fork?